src/combatsimulator/combatUnit.js

```diff
@@ -211,10 +211,11 @@ class CombatUnit {
         this.combatDetails.combatStats.physicalReflectPower += this.getBuffBoost(
             "/buff_types/physical_reflect_power"
         ).flatBoost;
-        this.combatDetails.combatStats.combatDropRate += this.getBuffBoost("/buff_types/combat_drop_rate").flatBoost;
         this.combatDetails.combatStats.combatExperience += this.getBuffBoost("/buff_types/wisdom").flatBoost;
         this.combatDetails.combatStats.criticalRate += this.getBuffBoost("/buff_types/crit").flatBoost;
         this.combatDetails.combatStats.criticalDamage += this.getBuffBoost("/buff_types/crit").flatBoost;
+
+        this.combatDetails.combatStats.combatDropRate = (1 + this.combatDetails.combatStats.combatDropRate) * this.getBuffBoost("/buff_types/combat_drop_rate").ratioBoost;
```

Comment:
Although currently no gear that increases drop rate exists, this will override it to 0 if you don't have a buff currently active. I recommend something along the lines of the following:       

`this.combatDetails.combatStats.combatDropRate += (this.combatDetails.combatStats.combatDropRate * this.getBuffBoost("/buff_types/combat_drop_rate").ratioBoost);`

Reply:
Good catch! That was supposed to be a `+=`. Though I do need to keep the +1, since combatDropRate/combatRareFind count up from 0.